fix(curriculum): Increased starting font size to fix a bug on Chrome browser for Sass Apply a Style Until a Condition is Met with @while challenge (#35935)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/03-front-end-libraries/sass/apply-a-style-until-a-condition-is-met-with-while.english.md

```yml
tests:
  - text: Your code should use the <code>@while</code> directive.
    testString: assert(code.match(/@while /g));

>   - text: Your code should set an index variable to 1 to start.

Reviewer: ```suggestion
  - text: Your code should start at an index of 1.
```
I think this sounds better to me. If you feel this is out of the scope, feel free to dismiss the review

Author: I am ok with the verbiage change, but I can not commit your suggestion, because you deleted a space before `- text:`

Reviewer: My bad, fixed it
